src/routes/lobby/LobbyView.vue

```diff
@@ -121,7 +119,14 @@ export default {
   setup() {
     const { t } = useI18n();
     const gameStarted = ref(false);
-
+    const joinAudio = ref(new Audio('/sounds/lobby/enter-lobby.mp3'));
```

Comment:
I don't think this needs to be a ref() since it shouldn't change, or really need reactivity

Reply:
Oh yes, I totally missed that.